app/web/src/features/messages/groupchats/GroupChatView.tsx

```diff
@@ -174,8 +175,15 @@ export default function GroupChatView() {
 
   const handleBack = () => history.goBack();
 
+  const title = groupChat
+    ? groupChatTitleText(groupChat, groupChatMembersQuery, currentUserId)
+    : groupChatError
+    ? "Error"
```

Comment:
Can this be a constant? I think reusing "Chat" or "Messages" is fine - they'll see the error on the page

Reply:
A constant? Do you mean to set a title to "Chat" or "Messages"? Now there will be name of user you are chatting with.

Reply:
Sorry - I meant specifically for the error case

Reply:
okay I simplified it. Hope I get it right